### PR TITLE
fix: Use column dep_snapshot to fetch SA manifests and remove reference to requestJson

### PR DESCRIPTION
**Pull request by rafiu007**
Since we are removing the requestJson column from the stack analyses table all references to it have to be removed.

### CODE REVIEW

**Review by deepak1725 — APPROVED** [review]
Just match the format of JSON saved in two columns respectively. 

Wait for other reviewers before the merge. Thanks

LGTM

File: f8a_report/helpers/report_helper.py
Comment (prashbnair) [inline f8a_report/helpers/report_helper.py:165]: Please make sure that this column in not referenced by name in the result anywhere in the code.

**Comment by rafiu007** [comment]
This pr is no longer applicable as we are keeping both columns dep_snapshot and requestJson.

**Pull request closed without merging**
